src/sentry/static/sentry/app/utils/discover/eventView.tsx

```diff
@@ -311,6 +314,7 @@ class EventView {
       yAxis: decodeScalar(location.query.yAxis),
       display: decodeScalar(location.query.display),
       interval: decodeScalar(location.query.interval),
+      expired: decodeScalar(location.query.expired) === 'true',
```

Comment:
I don't think we'll want to propagate this from the URL. This line could be safely removed since it is an optional parameter.